pkg/signature/kms/cliplugin/handler/options_test.go

```diff
@@ -0,0 +1,155 @@
+//
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package handler implements helper functions for plugins written in go.
+// It parses arguments  and return values to and from the supplied `SignerVerifier` implementation.
+
+//go:build !signer_program
```

Comment:
why does this have to be excluded?

Reply:
That's before I found a way to not include them with the `make test-signer-program` runs. I changed `./...` to `./`